### DISCUSSION TITLE
PCL II能塞进移动硬盘里吗？

### DISCUSSION THREAD

**Discussion by luotianci4751** [Meloong-Git/PCL#1953] [提问] [answered]
我比较好奇的是，PCL II可以塞进移动硬盘里吗？然后启动里面下载好的存档。（别问我为什么要问这个）

**Comment by so1ve** [comment, marked as answer]
可以，PCL 2本身就是一个可执行文件，你把他塞哪里都行

**Comment by a10086dd** [comment]
你把它塞mp3U盘一体的U盘都可以。

**Reply by iPodtouch6557** [reply]
换言之就是只要空间够并且数据没出错，塞哪都可以

**Reply by a10086dd** [reply]
知道

**Comment by a10086dd** [comment]
知道



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年06月10日 22:20 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [Hex-Dragon/PCL2] PCL II能塞进移动硬盘里吗？ (Discussion #1953) |

换言之就是只要空间够并且数据没出错，塞哪都可以

**Reply by iPodtouch6557** [reply]
刚处理完一个用邮件回无用信息的人，又来你重复回复……
请不要用邮件回复无用的信息，这样不仅阅览起来很麻烦，而且还增加一些解决问题负担

**Reply by a10086dd** [reply]
好的

**Reply by a10086dd** [reply]
> 刚处理完一个用邮件回无用信息的人，又来你重复回复……
> 请不要用邮件回复无用的信息，这样不仅阅览起来很麻烦，而且还增加一些解决问题负担

可是你不要发消息了吧，不然邮箱过一会就响一下

**Comment by Chiloven945** [comment]
点击旁边notifications的unsubscribe按钮取消订阅

**Reply by a10086dd** [reply]
我这是中文版的啊

**Reply by so1ve** [reply]
6

**Reply by a10086dd** [reply]
![Screenshot_2023-06-15-12-15-39-84_320a9a695de7cdce83ed5281148d6f19.jpg](https://github.com/Hex-Dragon/PCL2/assets/94156868/44df9af5-7580-4160-b7ff-26849d780aad)

你看我都不用网页版的👿

**Reply by so1ve** [reply]
那你就取消关注啊……

**Reply by a10086dd** [reply]
不会。

**Reply by iPodtouch6557** [reply]
不要讨论无关话题了，不讨论就没有如何unsubscribe的问题了……
